src/audio/load.c

```diff
@@ -375,17 +378,17 @@ void func_800BB43C(ALSeqFile *f, u8 *base) {
 
 void patch_sound(struct AudioBankSound *sound, u8 *memBase, u8 *offsetBase) {
     struct AudioBankSample *sample;
-    void *patched;
```

Comment:
`void *` should be used if the data is unknown. `u8*` is applicable if the data is a buffer of bytes. Or just a bunch of bytes.

For instances the actor casts in actors.c could arguably be a void * instead of Actor. Ex.
```c
void myActorUpdate(void * actor) {
   MyActor *myActor = (MyActor *) actor;
}
```